test/integration/features/profile.feature

```diff
@@ -2,89 +2,162 @@
 Feature: Profile
   As a user I can perform basic operations of Minishift with profile feature
 
-  Scenario: Starting Minishift with default profile
-     Given Minishift has state "Does Not Exist"
-      When executing "minishift start" succeeds
-      Then Minishift should have state "Running"
-
-Scenario: User should be able to list default profile 'minishift'
-   Given Minishift has state "Running"
-    When executing "minishift profile list" succeeds
-    Then stdout should contain
-    """
-    - minishift	Running		(Active)
-    """
-  Scenario: Getting default profile internal docker registry address
-     Given Minishift has state "Running"
-      When executing "minishift openshift registry" succeeds
-      Then stdout should be valid IP with port number
+  Scenario: Starting Minishift with default profile 'minishift'
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User should be able to list default profile 'minishift'
+    Given Minishift has state "Running"
+     When executing "minishift profile list" succeeds
+     Then stdout should match "-\s*minishift\s*Running\s*\(Active\)"
+
+  Scenario: Getting default profile 'minishift' internal docker registry address
```

Comment:
Just because it was there does not mean that it makes sense. We know the VM is running, it is in our pre-condition. If we want to check whether OpenShift is running we should test that and be explicit in what we want to assert. This assertion seems to be completely out of context for what we are trying to test.